<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Eai-ChandanQA" name="com.eaf.qa.uitests.BuilderTests" tests="1" failures="0" timestamp="23 Mar 2019 18:44:53 GMT" time="64.979" errors="1">
  <testcase name="testFeatureofRunningSection" time="64.979" classname="com.eaf.qa.uitests.BuilderTests">
    <error type="java.lang.AssertionError" message="Verify homepage is present expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Verify homepage is present expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at com.eaf.qa.actions.BaseActions.verify(BaseActions.java:87)
	at com.eaf.qa.uitests.BuilderTests.testFeatureofRunningSection(BuilderTests.java:565)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- testFeatureofRunningSection -->
</testsuite> <!-- com.eaf.qa.uitests.BuilderTests -->
